.github/workflows/deploy-dojo-for-e2e.yml

```
@@ -0,0 +1,316 @@
+  cancel-in-progress: false
+
+jobs:
+  build_images:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'automated-version-pr') }}
+    runs-on: ubuntu-latest
+    name: Build Images
+    strategy:
+      matrix:
+        app:
+          - {
+              project: "dojo-next",
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)